warehouse/query-core/src/main/java/datawave/query/tables/ssdeep/SSDeepScoringFunction.java

```diff
@@ -61,7 +63,15 @@ public class SSDeepScoringFunction implements Function<Map.Entry<Key,Value>,Stre
 
     private final SSDeepHashScorer<Set<NGramTuple>> ngramOverlapScorer;
 
-    public SSDeepScoringFunction(SSDeepSimilarityQueryConfiguration config) {
+    private final long maxResults;
+    private long totalResults = 0;
+
+    private final int maxHashesPerNGram;
+    private Map<String,Long> ngramCountMap;
+
+    public SSDeepScoringFunction(SSDeepSimilarityQueryConfiguration config, long maxResults, int maxHashesPerNGram) {
```

Comment:
Consider reading maxResults and maxHashesPerNGram from the config directly instead of passing additional arguments in?